src/css/application.css

```diff
@@ -0,0 +1,433 @@
+.underline-span {
+  position: relative;
+  display: inline-block;
+}
+
+.underline-span::after {
+  content: '';
+  position: absolute;
+  left: 0;
+  bottom: 0px;
+  height: 35%;
+  width: 100%;
+  background-color: var(--c-light-green);
+  transition: background-color 0.3s;
+  z-index: -1;
+  border-radius: 30px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.3);
+}
```

Comment:
to avoid code duplication pls remove this and just use existing in effects.css   class="underline-effect"